frontend/src/app/shared/components/editor/components/ckeditor-augmented-textarea/ckeditor-augmented-textarea.component.ts

```diff
@@ -288,23 +297,28 @@ export class CkeditorAugmentedTextareaComponent extends UntilDestroyedMixin impl
   }
 
   private setLabel() {
-    const label = document.querySelector<HTMLLabelElement>(`label[for=${this.textAreaId}]`)!;
-
     const ckContent = this.element.querySelector<HTMLElement>('.ck-content')!;
+    const label = document.querySelector<HTMLLabelElement>(`label[for=${this.textAreaId}]`);
 
-    let labelId;
-    if (label.hasAttribute('id')) {
-      labelId = label.getAttribute('id')!;
-    } else {
-      labelId = uniqueId('label-');
-      label.setAttribute('id', labelId);
-    }
+    if (label) {
+      let labelId;
```

Comment:
I may be nitpicking, but I prefer an early return in a guard clause rather than a long if.

```js
const label = document.querySelector<HTMLLabelElement>(`label[for=${this.textAreaId}]`);
if (!label) {
  console.error(`Please provide a label for the textarea with id ${this.textAreaId}`);
  return
}

...
```

That makes things clearer for me.

Reply:
Sure, I like guard clauses too. Will update 👍🏾 fwiw, I had an else if case before which is now removed, which is why the "original" structure is this way.